test/independent_mogp.jl

```diff
@@ -95,6 +95,11 @@
 
         # Ensure self-consistency.
         test_finitegp_primary_and_secondary_public_interface(rng, fx)
+
+        # Mix of by-features and by-outputs should also work efficiently.
+        y = MOInputIsotopicByOutputs(collect(range(0.0, 3.0; length=4)), 2)
+        test_internal_abstractgps_interface(rng, f, x, y)
```

Comment:
Are there any additional tests we want to run to ensure the correctness of `cov` in these cases?

Reply:
Yes. I'll check against naive computation. Good catch.